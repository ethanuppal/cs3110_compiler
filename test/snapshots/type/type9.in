if 1 + 2 { 
    let x = 5
}
