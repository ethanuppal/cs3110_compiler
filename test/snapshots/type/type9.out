Type error: in if statement condition: expected Bool but received Int
